torch/csrc/jit/mobile/function.cpp

```diff
@@ -65,10 +76,18 @@ void Function::set_register_size(size_t size) {
   code_->register_size_ = size;
 }
 
+std::string Function::get_module_debug_info(size_t pc) {
+  if (pc >= pc_to_module_debug_info_.size()) {
```

Comment:
Let's just assert here.

Reply:
Or rather assert if we have module debug info available but pc is larger.
And return "no module info" if debug info is unavailable.